assembl/static2/package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "test": "jest",
     "test:watch": "npm test -- --watch",
-    "build": "webpack --progress --colors",
+    "build": "rm -rf node_modules/react-i18nify/node_modules;webpack --progress --colors",
```

Comment:
does this re-download all react-i18nify dependencies everytime we update assembl's dependencies? (and have built it before)